img-txt_viewer.pyw

```diff
@@ -3726,7 +3727,12 @@ class ImgTxtViewer:
             else:
                 original_auto_save_var = self.auto_save_var.get()
                 self.auto_save_var.set(value=False)
-            directory = filedialog.askdirectory()
+            initialdir = self.image_dir.get()
+            if not initialdir or initialdir == "Choose Directory...":
+                initialdir = self.settings_manager.config.get("Path", "last_img_directory", fallback=None) 

```

Comment:
In your editor please enable `Trim Trailing Whitespace` to prevent saving unnecessary spaces etc. It just helps keep things cleaner.

If you use VS Code you can enable the setting: File → Preferences → Settings → User Settings → Search for: `Files: Trim Trailing Whitespace`

Reply:
> File → Preferences → Settings → User Settings → Search for: Files: Trim Trailing Whitespace

Why isn't this on by default lol, thank you